helm/templates/minio-pvc.yaml

```diff
@@ -2,10 +2,12 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: {{ .Release.Name }}-minio-pvc
+  namespace: qgnet
```

Comment:
In the README, I see that we say 

> Create a namespace called `qgnet` (or your preferred namespace).

However, here and in the `secrets.yaml`, we have the namespace hard-coded to `qgnet`.  Would the user need to edit these values to use a different namespace, or does this get changed dynamically?